### PR TITLE
Issue #533 - Lighter version of the show page

### PR DESCRIPTION
**Pull request by saniaio**
New designs for the show page that allows the metadata and description to be tucked away. New layout means the published item takes up much more space, with all of the details stowed away underneath. 
533_speech_revamp

### CODE REVIEW

File: client/src/components/VerticalCollapsibleSplit/index.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    this.state = { closed: false };
+  }
+
+  collapse () {
+    this.setState({ closed: !this.state.closed });
+    document.querySelectorAll(`[data-name='${this.props.name}']`).forEach(el => el.classList.toggle('closed'));
+  }
+
+  render () {
+    return (
```
Comment (skhameneh) [inline client/src/components/VerticalCollapsibleSplit/index.jsx]: you can use this before your ```return``` to clear up your references, it also has faster faster lookup when they're local vars:
```
let {
  props,
  state
} = this;
```

File: client/src/pages/ShowAssetDetails/view.jsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import PageLayout from '@components/PageLayout';
 
-import HorizontalSplit from '@components/HorizontalSplit';
-import AssetTitle from '@containers/AssetTitle';
+import VerticalCollapsibleSplit from '@components/VerticalCollapsibleSplit';
+// import VerticalSplit from '@components/VerticalSplit';
```
Comment (skhameneh) [inline client/src/pages/ShowAssetDetails/view.jsx]: Remove

File: public/assets/img/facebook.svg [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (skhameneh) [inline public/assets/img/facebook.svg]: Clean up, see the other comment on SVG files

File: client/src/components/VerticalCollapsibleSplit/index.jsx (hunk 1 above)
Comment (minesh93) [reply client/src/components/VerticalCollapsibleSplit/index.jsx]: Hi,
Could you elaborate on this? 
I've just implemented this but this is throwing an es-lint warning as its stating that those variables are unused?

Comment (skhameneh) [reply client/src/components/VerticalCollapsibleSplit/index.jsx]:
```
  render () {
    let {
      props,
      state,
    } = this;

    return (
      <div className={'vertical-split'}>
        <div className='visible-content' data-name={props.name}>
          {props.top}
          <button className='collapse-button' onClick={this.collapse}>
            {state.closed ? <Icon.PlusCircle className='plus-icon' />  : <Icon.MinusCircle /> }
          </button>
        </div>
        <div className='collapse-content' data-name={props.name}>
          {props.bottom}
        </div>
      </div>
    );
  }
```

**Pull request merged**